app/services/send_rabbitmq_message.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# Send a message to a RabbitMQ exchange that an item has been updated.
+class SendRabbitmqMessage
```

Comment:
This class is showing as entirely uncovered by tests. Are you thinking it doesn't need unit tests?

Reply:
Any test we could do here is merely going to be a mirror image of the code but in mocks.  I don't particularly thing that that sort of testing is useful.  Furthermore, we've tested this code already in qa and stage.

Reply:
Yeah, mocking the channel topic to test if it receives `#publish` with the args we expect does not seem worth the machinery or time.